Apps/W1/External File Storage - Azure File Service Connector/app/src/ExtFileShareAccountWizard.Page.al

```diff
@@ -0,0 +1,155 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.ExternalFileStorage;
+
+Using System.Environment;
+
+/// <summary>
+/// Displays an account that is being registered via the File Share connector.
+/// </summary>
+page 4571 "Ext. File Share Account Wizard"
+{
+    Caption = 'Setup Azure File Share Account';
+    SourceTable = "Ext. File Share Account";
+    SourceTableTemporary = true;
+    Permissions = tabledata "Ext. File Share Account" = rimd;
+    PageType = NavigatePage;
+    Extensible = false;
+    Editable = true;
+    ApplicationArea = All;
+
+    layout
+    {
+        area(Content)
+        {
+            group(TopBanner)
+            {
+                Editable = false;
+                ShowCaption = false;
+                Visible = TopBannerVisible;
+                field(NotDoneIcon; MediaResources."Media Reference")
+                {
+                    Editable = false;
+                    ShowCaption = false;
+                    ToolTip = ' ';
+                    Caption = ' ';
```

Comment:
If ShowCaption = false then you shouldn't have to set a Caption.
Furthermore the ToolTip here could be locked, since we don't want a translation here.